content/add-a-printer-ubuntu.md

```diff
@@ -51,6 +51,42 @@ sudo usermod -aG lpadmin $USER
 
 And when adding the printer, select type of connection as **DNS-SD**, and then **Forward**, and then **ESC/P-R**.
 
+## Brother Printers
+
+Brother provides a driver installation tool for Linux users. Install the appropriate driver for your Brother printer by downloading this tool and running the installer with your printer model appended to the command.
+
+1. Click [this link](https://support.brother.com/g/b/productsearch.aspx?c=us&lang=en&content=dl) to download Brother's Driver Install Tool for Linux while searching for your appropriate printer.
```

Comment:
I believe this should be "for the appropriate printer"

Reply:
I think either works, but I think "the appropriate" sounds better.

Reply:
I would agree